### PR TITLE
Sticky header

### PR DESCRIPTION
**Pull request by buberdds**

### CODE REVIEW

**Comment by csillag** [comment]
I tested this and it works great, but I'm a bit concerned by the fact that the composition of the components is different on desktop and mobile. (Ie. the search is included in PageLayou on desktop, but in Header on mobile.)

While I see why this might have been the most natural way to implement this now from a technical point of view, I am concerned that maintaining this in the future might be difficult and error prone, because there are too many variations. 

Wouldn't it be possible to avoid this fundamental difference between desktop and mobile, somehow?

File: src/app/components/PageLayout/Logotype.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { useTranslation } from 'react-i18next'
 
 interface LogotypeProps {
   color?: string
-  showText: boolean
+  showText?: true | false | undefined // If undefined: shows text on desktop
```
Comment (csillag) [inline src/app/components/PageLayout/Logotype.tsx:12]: Do we need both the question mark _and_ the undefined value? Seems a bit redundant...

Comment (lukaw3d) [reply src/app/components/PageLayout/Logotype.tsx:12]: Nope

**Comment by lukaw3d** [comment]
Rebased and split commits

**Review by lukaw3d — APPROVED** [review]

**Comment by csillag** [comment]
> [lukaw3d](https://github.com/lukaw3d) approved these changes [2 minutes ago](https://github.com/oasisprotocol/explorer/pull/466#pullrequestreview-1470950128)

I think since @buberdds is not around ATM, we should just merge this, now that it's approved.

**Pull request merged**